orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/OortService.groovy

```diff
@@ -31,12 +31,18 @@ interface OortService {
                       @Path("cloudProvider") String cloudProvider)
 
   @GET("/applications/{app}/clusters/{account}/{cluster}/{cloudProvider}/serverGroups/{serverGroup}")
+  Response getServerGroupFromCluster(@Path("app") String app,
```

Comment:
The worst part of this particular API is that the Response could be:

- a Map if the `region` query parameter is supplied
- a Collection if the `region` query parameter is NOT supplied

Both forms are being called.